### DISCUSSION TITLE
Get the exact list of attributes/methods checked by `@runtime_checkable` decorated `Protocol` class.

### DISCUSSION THREAD

**Discussion by randolf-scholz** [python/typing#1470] [Q&A] [answered]
For a given `Protocol` class, how can one get the exact list of all the things defined by that `Protocol`? The list should be identical with the list of attributes/methods `@runtime_checkable` tests if the `Protocol` is decorated with it.

**Comment by Gobot1234** [comment, marked as answer]
Python 3.13 added a function to help, with this https://docs.python.org/3.13/library/typing.html#typing.get_protocol_members

**Reply by randolf-scholz** [reply]
@Gobot1234 Ah nice, it also in typing_extensions 4.7.0
